doc/lua_api.md

```diff
@@ -291,6 +291,43 @@ in one of its parents, the parent's file is used.
 Although it is discouraged, a mod can overwrite a media file of any mod that it
 depends on by supplying a file with an equal name.
 
+Of the model file formats, only a subset is supported:
+
+Simple textured meshes (with multiple textures), optionally with normals.
+The .x and .b3d formats additionally support skeletal animation.
```

Comment:
A bit hard to understand imo.
Only a subset of the model file formats is supported?
Maybe only a subset of the features supported?

Reply:
https://github.com/minetest/minetest/pull/14557/commits/aed59bb2ba408c3177f363c46a06e3f2d98fc1e1